### PR TITLE
radare/radare2 issue #4173 - initial fix

### PR DESCRIPTION
**Pull request by jarusified**

### CODE REVIEW

File: r2rap/python/remote.py [hunk 1]
```diff
@@ -140,21 +144,45 @@ def _handle_client(self, c):
...
 				break
 
+	def listen_stdin(self):
+		line = sys.stdin.readline()
+
+		def threaded_client(line):
+			cl = RapClient("localhost", 9999)
+			cl.cmd(line)
+
+		threading.Thread(target=threaded_client, args=(line.strip(), )).start()
```
Comment (radare) [inline r2rap/python/remote.py]: i would prefer not to create threads from this library, this can be delegated to the user who uses the api.

**Comment by jarusified** [comment]
yes ofcourse, i did this just to test the working.

**Comment by radare** [comment]
Is threading necessary?

**Comment by jarusified** [comment]
Nopes, not required. Instead of writing client API directly, I`` thought we could test with the already existing API. So I create a separate instance of client everytime there is an input to the stdin and disconnect once it is parsed.

**Comment by radare** [comment]
it will be cool to package the rap.py api into pip, as its already done for r2pipe too

**Comment by radare** [comment]
ping

**Comment by radare** [comment]
ping

**Comment by radare** [comment]
timeout

**Pull request closed without merging**